eng/common/scripts/Verify-Links.ps1

```diff
@@ -115,15 +117,19 @@ function ParseLinks([string]$baseUri, [string]$htmlContent)
 function CheckLink ([System.Uri]$linkUri)
 {
   if ($checkedLinks.ContainsKey($linkUri)) { return }
+  $badLinkEnabled = $false;
```

Comment:
I've been wanting to update this function to return true if the link is valid and false if the link is not valid. So can you instead rename this to `$linkValid = $true` and then set it to false anytime we detect it is bad? Then update the $checkedLinks dictionary to store the value and return it at the top of the function and return `$linkValid` at the end of this function.

Reply:
Fixed as you suggested, much better for the entire structure.
$checkedLinks currently stores like : {<link>: <valid or not>}